src/app/app-routing.module.ts

```diff
@@ -19,7 +21,11 @@ export class Activate implements CanActivate {
 
 
 const routes: Routes = [
-  {path: '', component: MainComponent, canActivate: [Activate]},
+  {path: '', component: MainComponent, canActivate: [Activate], children: [
+    {path: '', component: MainWelcomeComponent, canActivateChild: [Activate]},
+    {path: 'project/:projectUUID', component: MainProjectComponent},
```

Comment:
Going directly to a route like`/project/4baf3a88-c484-4a60-9f33-9862064fab62` doesn't work at the moment.  Maybe bets to work on that after this PR.  Suggestion:  let's merge this PR and either fix that separately or potentially handle that direct route in https://github.com/TACC-Cloud/hazmapper/pull/38

Reply:
We'll look at this later in #38 or https://jira.tacc.utexas.edu/browse/DES-1894 (👍  on creating ticket)